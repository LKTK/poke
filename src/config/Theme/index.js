import { ThemeProvider  } from '@material-ui/core/styles'
import theme from 'src/config/Theme/theme.config';

const LonPokeThemeProvider = ({ children }) => {
    return (
        <ThemeProvider theme={theme}>
            {children}
        </ThemeProvider>
    )
}  

export default LonPokeThemeProvider