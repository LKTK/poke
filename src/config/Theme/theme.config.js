import { createMuiTheme } from '@material-ui/core/styles'

const theme = createMuiTheme({
    palette: {
      primary: {
        main: '#ffcb05',
        dark: '#b38e04',
        light: 'e6b705',
      },
      secondary: {
        main: '#3d7dca',
        dark: '#003a70',
        light: '#518acf',
      },
    },
  });

export default theme;