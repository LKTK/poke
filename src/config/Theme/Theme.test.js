import React from 'react'
import TestRenderer from 'react-test-renderer'
import Theme from 'src/config/Theme'

describe('src/config/Theme', () => {

    it('should render the theme provider', () => {
        const testRenderer = TestRenderer.create(<Theme><span>TEST THEME</span></Theme>);
        expect(testRenderer).toMatchSnapshot();
    })

})