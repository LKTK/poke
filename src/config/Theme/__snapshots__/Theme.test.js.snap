// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/config/Theme should render the theme provider 1`] = `
<span>
  TEST THEME
</span>
`;
